src/LibVLCSharp/Shared/Core/Core.cs

```diff
@@ -57,7 +59,7 @@ static void Log(string message)
         }
 
 #if (NETFRAMEWORK || NETSTANDARD) && !NETSTANDARD1_1
-        static bool Loaded => LibvlcHandle != IntPtr.Zero;
+        static bool DesktopLoaded => LibvlcHandle != IntPtr.Zero;
```

Comment:
`LibraryLoaded` or `NativeLibraryLoaded` or `LibVLCLoaded` instead ?

Reply:
Yep, simplified the libvlc loading across targets (there were 2, desktop where we have a native handle and others) and took into account your renaming remark.